core/controllers/access_validators_test.py

```
@@ -1775,3 +1775,43 @@ def test_get_fails_when_story_doesnt_exist(self) -> None:
+        self.signup(self.TECH_LEAD_EMAIL, self.TECH_LEAD_USERNAME)
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+
+        self.add_user_role(
+            self.TECH_LEAD_USERNAME,
+            feconf.ROLE_ID_TECH_LEAD,
+        )
+
+    def test_guest_user_does_not_pass_validation(self) -> None:
+        self.get_json(
+            '%s/can_access_technical_feedback_dashboard'
+            % ACCESS_VALIDATION_HANDLER_PREFIX,
```

Ruff (0.15.20)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)